src/entity/EnemyShipSet.java

```diff
@@ -30,17 +30,23 @@ public class EnemyShipSet {
     private double X_speed = 1.0;
     // 적 함선의 Y 방향 속도
     private double Y_speed = 1.0;
+
+    private int base_hp = 2;
```

Comment:
```suggestion
    private int base_hp;
```
아래 setEnemyHp() 메소드를 통해 적군의 초기 hp를 설정하므로 따로 초기 값 할당은 필요 없을 것 같습니다. 
추가로 변수에 대한 주석 추가하면 좋을 것 같습니다.